### DISCUSSION TITLE
Change order of printer display in 'current operations' screen

### DISCUSSION THREAD

**Discussion by pat1121** [OctoFarm/OctoFarm#742] [Feature Request]
Currently my printers are arranged by completion percentage. I'd much prefer to have them arranged by the same order I have them in the 'file upload' menu. Or to be able to move them around?

A dropdown menu so they could be rearranged at will would be ideal! Perhaps:

- by completion percentage
- by print time remaining (ascending, descending)
- by pre-arranged order (like the file manager)
- by file name (that they are currently printing)

**Comment by NotExpectedYet** [comment]
Agree with you here as it's something I've wanted to add for a while but never gotten around to! Thanks for adding the request.

**Comment by NotExpectedYet** [comment]
> Currently my printers are arranged by completion percentage. I'd much prefer to have them arranged by the same order I have them in the 'file upload' menu. Or to be able to move them around?
> 
> A dropdown menu so they could be rearranged at will would be ideal! Perhaps:
> 
> * by completion percentage
> * by print time remaining (ascending, descending)
> * by pre-arranged order (like the file manager)
> * by file name (that they are currently printing)

I will be enabling these sorting options for V1.2 :) Thanks again for the reminder! 

sortIndex is the index you refer to in file manager. There's also progress (current), time remaining and file name. 

![image](https://user-images.githubusercontent.com/61137988/138460940-37a79f80-0f2d-46ed-abb4-ff6fe639f04d.png)
